crowbar_framework/app/models/node_object.rb

```diff
@@ -259,6 +260,7 @@ def initialize(node)
     end
     # deep clone of @role.default_attributes, used when saving node
     @attrs_last_saved = @role.default_attributes.deep_dup
+    @attributes_changed_for_event = Set.new
```

Comment:
As discussed could you rename this to `@attributes_changed` and then add a comment explaining what it's used for?